spec/models/organisation_spec.rb

```diff
@@ -170,7 +170,7 @@ def build_org_with_computed_fields_and_updated_at org, updated_at = nil
     end
     it 'uses org admin mailer to email existent user when upgraded to org admin' do
       usr = FactoryGirl.create(:user, :email => 'user@example.org')
-      mockMessage = double("message")
+      mockMessage = double('message')
```

Comment:
It isn't my change tansaku ^^.

2016-05-09 13:14 GMT+02:00 Hound <notifications@github.com>:

> In spec/models/organisation_spec.rb
> <https://github.com/AgileVentures/LocalSupport/pull/304#discussion_r62482609>
> :
>
> > @@ -170,7 +170,7 @@ def build_org_with_computed_fields_and_updated_at org, updated_at = nil
> >      end
> >      it 'uses org admin mailer to email existent user when upgraded to org admin' do
> >        usr = FactoryGirl.create(:user, :email => 'user@example.org')
> > -      mockMessage = double("message")
> > +      mockMessage = double('message')
>
> Style/VariableName: Use snake_case for variable names. (
> https://github.com/bbatsov/ruby-style-guide#snake-case-symbols-methods-vars
> )
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/AgileVentures/LocalSupport/pull/304/files/45c060a7110d7332b97860fd80dbec7fc0b86f48#r62482609>
>